### ISSUE TITLE
Hebergement Sitra parser displayed twice in admin

### ISSUE THREAD

**Issue by camillemonchicourt** [makinacorpus/Geotrek#1600]
Due to Sitra parsers improvements ? 
https://github.com/makinacorpus/Geotrek/issues/1482

![parser](https://cloud.githubusercontent.com/assets/4418840/16198001/7c717cea-3704-11e6-84d9-c600426884db.jpg)

Displayed twice while it is only once in the ``bulkimport/parsers.py``.
Only the second one is working.

```
# -*- coding: utf8 -*-

from geotrek.tourism.parsers import HebergementsSitraParser

class HebergementParser(HebergementsSitraParser):
    api_key = '4Vcrxxxx'
    project_id = 15xx
    selection_id = 370xx
    source = ['Apidae']
```

**Comment by submarcos** [comment]
OK, so there is a bug. Parsers shown in page are those with a label.
HebergementsSitraParser have a label, so it's displayed.

You override HebergementsSitraParser with HebergementParser without override its label.

So you have 2 parsers with same label.

Solution : define a custom label in HebergementParser. And wait a bugfix which delete baseparsers from list.

```
class HebergementParser(HebergementsSitraParser):
    label = "My Custom Parser Hebergements"
    ****
```

**Comment by camillemonchicourt** [comment]
OK I will try that asap.

**Comment by camillemonchicourt** [comment]
I am trying to update my parser, based on actual master and the parser is only displayed once. 

But I get this warning for each object, when importing : 

```
126 avertissements:
# Line 18:
- Type De Contenu Touristique 'G' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'î' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 't' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'e' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique '' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'd' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique ''' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'é' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 't' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'a' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'p' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'e' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique '/' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 's' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'é' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'j' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'o' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'u' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'r' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
# Line 19:
- Type De Contenu Touristique 'G' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'î' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 't' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'e' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique '' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'd' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique ''' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'é' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 't' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'a' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'p' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
- Type De Contenu Touristique 'e' n'existe pas dans Geotrek-Admin. Merci de l'ajouter,
```

I don't know if it is due to recent changes ?

**Comment by submarcos** [comment]
I think that "Type de Contenu Touristique" is a string, and should be an Array of string.

just replace: 
`u"Gîte d'étape"`
by : `[u"Gîte d'étape", ]`

**Comment by camillemonchicourt** [comment]
OK but I dont see where I could replace it. 
"Gîte d'étape / séjour" is just one of the values in my `tourisme.t_b_contenu_touristique_type` table. 
And also in SITRA datas.

**Comment by submarcos** [comment]
add method in your parser : 

```
    from geotrek.tourism.models import TouristicContentType

    ********

    class YourParser(BaseParser):
    ****
    def filter_type1(self, src, val):
        type1_val, created = (TouristicContentType.objects.get_or_create(label=val, in_list=1, category=self.obj.category))
        return [type1_val]
```

**Comment by gutard** [comment]
> the parser is only displayed once.

So the bug is fixed. Please close this issue.

> But ...

If there is an **other** problem, please open an **other** issue.
Otherwise we get confused because the title of the issue does not match the current problem.

**Comment by camillemonchicourt** [comment]
OK right, I created another issue for the second problem - https://github.com/makinacorpus/Geotrek/issues/1613

Thanks
